src/spec/payid-protocol.md

```diff
@@ -491,11 +483,11 @@ This specification defines the PayID protocol - an application-layer protocol, w
 ## Information Integrity
   The HTTPS connection provides transport security for the interaction between PayID client and server but does not provide the response integrity of the data provided by PayID server. A PayID client has no way of knowing if data provided in the payment account information resource has been manipulated at the PayID server, either due to malicious behavior on the part of the PayID server administrator or as a result of being compromised by an attacker. As with any information service available on the Internet, PayID clients should be wary of the information received from untrusted sources.
 
-# Privacy Considerations
+# Privacy Considerations (Best Practices)
```

Comment:
Should this primary title be `Best Practices` with `Privacy Considerations` being a secondary heading?

In other words, could `Privacy Considerations`, `Access Control`, `Payment Address Rotation` etc all be grouped into "Best Practices"?